package POOAULA3.exceptions;
/* Exercício 2 - Calculadora Segura

1. Na classe Main, defina duas variáveis int:
○ numeroA = 10
○ numeroB = 0
2. Crie um bloco try para envolver a operação de divisão: int resultado = numeroA /
numeroB;.
3. Imprima o resultado dentro do bloco try (esta linha não será alcançada).
4. Crie um bloco catch que capture a exceção ArithmeticException.
5. Dentro do bloco catch, imprima uma mensagem amigável: "Erro detectado! Não
é possível realizar uma divisão por zero."
6. Adicione um bloco finally que imprima: "Execução da calculadora encerrada." */


public class CalculadoraSegura {
    
}
